manifests/pipecd/templates/configmap.yaml

```diff
@@ -9,3 +9,151 @@ data:
   {{ .Values.config.fileName }}: |-
 {{ tpl .Values.config.data . | indent 4 }}
 {{- end }}
+
+{{- if .Values.monitoring.enabled -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "pipecd.fullname" . }}-grafana-dashboards
+  labels:
+    {{- include "pipecd.labels" . | nindent 4 }}
+    grafana_dashboard: "1"
+data:
+  # TODO: Make dashboard for application metrics
+  application-metrics.json: |-
+    {
```

Comment:
Instead of embedding the dashboard files into this ConfigMap like this, let make a `dashboards` directory to store all of them and load them to this ConfigMap. All dashboard files can just be exported from Grafana and pasted into that directory.

Reply:
Obviously it's better to do so! What you're suggesting is more manageable and honestly it could be a little bit tedious to care about indent in configmap.yaml.